crypto/rand/rand_lib.c

```diff
@@ -210,13 +210,14 @@ size_t rand_drbg_get_additional_data(unsigned char **pout, size_t max_len)
     size_t len;
 #ifdef OPENSSL_SYS_UNIX
     pid_t pid;
-    struct timespec ts;
+    struct timeval tv;
 #elif defined(OPENSSL_SYS_WIN32)
     DWORD pid;
     FILETIME ft;
     LARGE_INTEGER pc;
-#endif
     uint32_t tsc = 0;
+#endif
```

Comment:
Not quite, and sorry for not pointing it out...  judging from the rest of the code, it should be guarded with `#ifdef OPENSSL_CPUID_OBJ`

Reply:
Yeah, sorry. I got interrupted in the middle of doing it, so I was distracted a little.